.github/workflows/prepare-release.yml

```diff
@@ -156,25 +149,26 @@ jobs:
       - name: Temporary tag
         run: git tag "$INPUT_TAG"
 
-      - name: Set up Java
-        # FIXME: v4+ requires Node 20
-        uses: actions/setup-java@v3
-        with:
-          java-version: "11"
-          distribution: "temurin"
+      - name: Build native test server (non-Docker)
+        if: matrix.os_family != 'Linux'
+        run: |
+          ./gradlew -x :temporal-serviceclient:updateSubmodules :temporal-test-server:build
 
-      - name: Set up Gradle
-        # FIXME: v3+ requires Node 20
-        uses: gradle/gradle-build-action@v2
-      
-      - name: Build native test server
-        run: ./gradlew :temporal-test-server:build
+      - name: Build native test server (Docker)
+        if: matrix.os_family == 'Linux'
+        run: |
+          docker run \
+            --rm -v "$(pwd):/workspace" -w /workspace \
+            ghcr.io/graalvm/graalvm-community:21-ol7 \
```

Comment:
Also the graal compiler should come from our plugin not the environment so why did you need to use a graal docker image here?

Reply:
Replaced by `openjdk:21-jdk-oraclelinux7`.